documentation/DEV_External_Software.md

```diff
@@ -0,0 +1,9 @@
+# External Software (Developers edition)
+
+Vita3K uses multiple pieces of external software. The main five are Dynarmic, Unicorn, SDL, OpenGL and Vulkan.
+
+# Dynarmic
+
+Notice that Dynarmic is Dyn**arm**ic and not Dyn**am**ic. This is because Dynarmic is an ARM interpreter. All it does is converts low-end ARM instructions to high-end, executable instructions for the CPU. It keeps the emulator fast without having to emulate every little bit of the CPU, which is a major upside to Vita3K.
+
+Before we continue, you can look at the Dynarmic GitHub repo by clicking [here](https://example.com/).
```

Comment:
https://github.com/merryhime/dynarmic

Reply:
thank you! I had it on my clipboard and didnt even paste it in